Sources/MapboxNavigation/MapView.swift

```diff
@@ -174,33 +174,94 @@ extension MapView {
         
         return datasets
     }
-
-    var mainRouteLineParentLayerIdentifier: String? {
-        var parentLayer: String? = nil
-        let identifiers = [
+    
+    func layerPosition(for layerIdentifier: String, route: Route? = nil, customLayerPosition: LayerPosition? = nil) -> LayerPosition? {
```

Comment:
Should this method be moved to a NavigationMapView extension? It’s only called from a NavigationMapView extension method and makes a lot of references to NavigationMapView. I’m not sure that the method would be very useful for a standalone MapView. If we move it to NavigationMapView, then `NavigationMapView.LayerIdentifier` simplifies to just `LayerIdentifier`.